### PR TITLE
Adjust tests to start from a new Url instance each time, fix double equal bug

### PR DESCRIPTION
**Pull request by m4olivei**
Was seeing an issue where the `autoPlay=false` query string param wouldn't show in the player URL generated. Odd b/c there was a test that seemed to show it was working fine. Turns out the test setup was showing a false positive.

### CODE REVIEW

File: src/Service/Player/Url.php [hunk 1]
```diff
@@ -148,7 +148,7 @@ public function __toString()
      */
     public function withAutoplay(bool $autoPlay): self
     {
-        if ($this->autoPlay == $autoPlay) {
+        if ($this->autoPlay === $autoPlay) {
```
Comment (deviantintegral) [inline src/Service/Player/Url.php:151]: I don't see a test covering this specifically? I would have expected `$this->autoPlay` to somehow be set to a non-boolean.

**Review by deviantintegral — APPROVED** [review]
A question, but you're more in touch this this code than I am. Merge at will!

File: src/Service/Player/Url.php (hunk 1 above)
Comment (m4olivei) [reply src/Service/Player/Url.php:151]: The trouble was that when `$this->autoPlay === NULL`, when you try to set `$url->withAutoplay(FALSE)` this conditional with a double equal would evaluate to `TRUE` and then `$this->autoPlay` would remain as `NULL`. The tests as they were were hiding this bug b/c the tests would build on a single instance of a `Url` object, first testing `$url->withAutoplay(TRUE)`, which would indeed set the value for `$this->autoPlay`, then follow that assertion by setting `$url->withAutoplay(FALSE)`, which would indeed set the value for `$this->autoPlay` to `FALSE`, behaving as expected.

In short, this would work:

```
$url = new \Lullabot\Mpx\Service\Player\Url();
$url = $url->withAutoPlay(TRUE);
$url = $url->withAutoPlay(FALSE);
$url_string = (string) $url;
/// $url_string has &autoPlay=false
```
But this wouldn't work:
```
$url = new \Lullabot\Mpx\Service\Player\Url();
$url = $url->withAutoPlay(FALSE);
$url_string = (string) $url;
/// $url_string is missing &autoPlay=false. sad panda.
```
I changed the tests so that each assertion acts on a new instance.
